test/e2e/pkg/infra/digitalocean/digitalocean.go

```diff
@@ -47,9 +52,32 @@ func (p Provider) StartNodes(ctx context.Context, nodes ...*e2e.Node) error {
 	return execAnsible(ctx, p.Testnet.Dir, playbookFile, nodeIPs)
 }
 
-// Currently unsupported.
-func (p Provider) SetLatency(_ context.Context, _ *e2e.Node) error {
-	return fmt.Errorf("SetLatency() currently unsupported for Digital Ocean")
+// SetLatency prepares and executee the latency-setter script in the given node.
+func (p Provider) SetLatency(ctx context.Context, node *e2e.Node) error {
+	// Directory in the DigitalOcean node that contains all latency files.
+	remoteDir := "/root/cometbft/test/e2e/latency/"
+
+	playbook := "- name: e2e custom playbook\n" +
+		"  hosts: all\n" +
+		"  tasks:\n"
```

Comment:
```suggestion
	playbook := `- name: e2e custom playbook
  hosts: all
  tasks:
`
```

Reply:
Indentation in the ansible playbook should be flawless, otherwise they won't run. I found that having each line enclosed with quotes is better to keep the indentation correct.